src/metaprob/to_clojure.clj

```diff
@@ -225,13 +225,13 @@
 (defn declarations [subs]
   (assert (seq? subs))
   (let [defs (filter gen-definition? subs)]
-    (do (assert (seq? defs) "no brainer")
+    (assert (seq? defs) "no brainer")
```

Comment:
I think I'd just delete this, I don't see any way `filter` could return a non-seq.  This is vestigial from some debugging session.